core/domain/question_domain_test.py

```diff
@@ -19,12 +19,48 @@
 from core.domain import exp_domain
 from core.domain import question_domain
 from core.tests import test_utils
+import feconf
 import utils
 
 
 class QuestionDomainTest(test_utils.GenericTestBase):
     """Tests for Question domain object."""
 
+    def _create_valid_question_data(self, default_dest_state_name):
```

Comment:
Why duplicate? If you need a setup method common to both tests, then put it in core/tests/test_utils.py. In general try not to duplicate code.

Reply:
Done, moved the function to test_utils.